### PR TITLE
Add note on how to test

### LINKED ISSUE #104: Debian python3.9 test suite fails
# Description

Test suite 

## Steps for reproduction

Run this command in the flict dir;
`$  python3.9 -m pytest tests`

## Expected behavior
Successful execution of test suite.

## Actual behavior
```
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --bandit --random-order --random-order-bucket=global 
  inifile: /home/jeremiah/Packaging/clean/flict/setup.cfg
  rootdir: /home/jeremiah/Packaging/clean/flict  
```
## Environment

 | tool    | version    | how to get it       |
 | ------- | ---------- | ------------------- |
 | flict   | 91-move-makefile branch | `flict --version`   |
 | python3 | 3.9.7 | `python3 --version` |
 | os      | Debian testing | n.a.                |

Using flict branch  on Debian testing

## Additional logs

Removing `--random-order --random-order-bucket=global --bandit` fixes the error. This *may* be due to missing python test packages in Debian.

### PR DESCRIPTION
**Pull request by hesa**
Short note on how to test flict is added to CONTRIBUTING.md

Fixes #104

### CODE REVIEW

**Comment by hesa** [comment]
Failures (build) fixed with #144 
We can discard the errors

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,5 +10,20 @@ signing off?
...
 
+# Testing
+
+To execute test flict:
+
+```
+$ python3 -m pytest tests
+```
+
+and optionall do:
```
Comment (jens-erdmann) [inline CONTRIBUTING.md]: Typo: `optional`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,5 +10,20 @@ signing off?
 
 # Governance
 
+# Testing
+
+To execute test flict:
+
+```
+$ python3 -m pytest tests
```
Comment (priv-kweihmann) [inline CONTRIBUTING.md]: Normally just running `pytest` is enough

**Comment by priv-kweihmann** [comment]
This branch needs a rebase but then should be good to go

**Comment by jens-erdmann** [comment]
> This branch needs a rebase but then should be good to go

Took the liberty to get this done.

**Pull request merged**